app/models/tasks/copy_media_task.rb

```diff
@@ -1,15 +1,18 @@
 class Tasks::CopyMediaTask < ::Task
   before_save do
     if media_resource && status_changed?
-      media_resource.update!(status: status)
```

Comment:
I was hitting a bug, where the "processing" view would switch to "complete" too early.  Before the audio metadata (duration, bitrate, etc) was there.

Turns out - this method was making 2 different `update!` calls.  Refactored a bit so it only makes 1.  Otherwise equivalent.